# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/xre/nsEmbedFunctions.cpp
# Commit: ef46ec074d6a
# Full Hash: ef46ec074d6aba10d6e4342e305f36ead6996ddb
# Author: Honza Bambas <honzab.moz@firemni.cz>
# Date: 2018-06-07 21:50:48
# Regressor Bug: 1413868
# File Overlap Count: 1
# Description:
#   Bug 1413868. r=valentin
# ==============================================================================

diff -r 14f7dab3a9bd -r ef46ec074d6a toolkit/xre/nsEmbedFunctions.cpp
--- a/toolkit/xre/nsEmbedFunctions.cpp	Wed Jun 06 14:50:17 2018 -0700
+++ b/toolkit/xre/nsEmbedFunctions.cpp	Thu Jun 07 13:56:16 2018 -0400
@@ -58,6 +58,7 @@
 #endif //  defined(MOZ_WIDGET_ANDROID)
 
 #include "mozilla/AbstractThread.h"
+#include "mozilla/FilePreferences.h"
 
 #include "mozilla/ipc/BrowserProcessSubThread.h"
 #include "mozilla/ipc/GeckoChildProcessHost.h"
@@ -708,6 +709,8 @@
       // InitLoggingIfRequired may need access to prefs.
       mozilla::sandboxing::InitLoggingIfRequired(aChildData->ProvideLogFunction);
 #endif
+      mozilla::FilePreferences::InitDirectoriesWhitelist();
+      mozilla::FilePreferences::InitPrefs();
 
       OverrideDefaultLocaleIfNeeded();
 